decks-support-store/templates/support-store-stateful-set.yaml

```
@@ -0,0 +1,101 @@
+{{- if ne .Values.global.platform "VMware"}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: decks-support-store-headless
```

```suggestion
  name: decks-support-store
```
do we need `headless`  we don't have that for the deployment.

---

no..it has to be headless 
we have another service without headless